components/list/examples/and_list.rs

```diff
@@ -6,12 +6,15 @@
 icu_benchmark_macros::instrument!();
 use icu_benchmark_macros::println;
 
-use icu::list::{ListFormatter, ListLength};
+use icu::list::{ListFormatter, ListFormatterOptions, ListLength};
 use icu::locale::locale;
 
 fn main() {
-    let list_formatter =
-        ListFormatter::try_new_and_with_length(&locale!("es").into(), ListLength::Wide).unwrap();
+    let list_formatter = ListFormatter::try_new_and_with_length(
+        locale!("es").into(),
+        ListFormatterOptions::new().style(ListLength::Wide),
```

Comment:
this should probably be `with_style`

Reply:
"style" is discouraged; should be `with_length`

(this is moot if we revert `ListFormatterOptions`)